stores/sql/sqlite/migrations/main/migration_00018_gouging_units.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE settings
```

Comment:
Have you tested this in SQLite? I don't think we have `JSON_UNQUOTE`, and `JSON_REPLACE` was introduced in 2022, not sure what min version of SQLite we want to support. This is one of the reasons why I wanted to do the settings migration from code instead of writing it out in raw SQL